src/app/components/Enable/LiquidEnable.tsx

```diff
@@ -0,0 +1,122 @@
+import { CheckIcon } from "@bitcoin-design/bitcoin-icons-react/filled";
+import ConfirmOrCancel from "@components/ConfirmOrCancel";
+import Container from "@components/Container";
+import PublisherCard from "@components/PublisherCard";
+import { useCallback, useEffect, useRef, useState } from "react";
+import { useTranslation } from "react-i18next";
+import ScreenHeader from "~/app/components/ScreenHeader";
+import toast from "~/app/components/Toast";
+import { USER_REJECTED_ERROR } from "~/common/constants";
+import msg from "~/common/lib/msg";
+import type { OriginData } from "~/types";
+
+type Props = {
+  origin: OriginData;
+};
+function LiquidEnableComponent(props: Props) {
+  const hasFetchedData = useRef(false);
+  const [loading, setLoading] = useState(false);
+  const { t } = useTranslation("translation", {
+    keyPrefix: "enable",
+  });
+  const { t: tCommon } = useTranslation("common");
+
+  const enable = useCallback(() => {
+    try {
+      setLoading(true);
+      msg.reply({
+        enabled: true,
+        remember: true,
+      });
+    } catch (e) {
+      console.error(e);
+      if (e instanceof Error) toast.error(`${tCommon("error")}: ${e.message}`);
+    } finally {
+      setLoading(false);
+    }
+  }, [tCommon]);
+
+  function reject(event: React.MouseEvent<HTMLAnchorElement>) {
+    event.preventDefault();
+    msg.error(USER_REJECTED_ERROR);
+  }
+
+  async function block(event: React.MouseEvent<HTMLAnchorElement>) {
```

Comment:
I have seen time and time again when we have 5+ duplicated files like this, changes are incorrectly propagated across all of them and this leads to bugs. @bumi I know you said just copy these screens, but all the code for handling allowances/blocklists is also duplicated.

Reply:
it's just a function call, there is no logic. 
I think you add more logic and complexity to the code when you try to unify it.